src/themes/capybara/pages/Home.vue

```diff
@@ -0,0 +1,273 @@
+        <SfCarouselItem v-for="(product, i) in newProducts" :key="i">
+          <SfProductCard
+            :title="product.name"
+            :regular-price="product.priceInclTax | price"
+            class="product-card"
+          />
+        </SfCarouselItem>
+      </SfCarousel>
+    </SfSection>
+    <SfSection
+      :heading="{
+        title: 'Share Your Look',
+        subtitle: '#YOURLOOK'
+      }"
+    >
+      <div class="grid grid-images">
+        <div class="grid__row">
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageA.png">
+              katherina_trn
+            </SfImage>
+          </div>
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageB.png">
+              katherina_trn
+            </SfImage>
+          </div>
+        </div>
+        <div class="grid__row">
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageC.png">
+              katherina_trn
+            </SfImage>
+          </div>
+          <div class="grid__col">
+            <SfImage src="/assets/homepage/imageD.png">
+              katherina_trn
+            </SfImage>
+          </div>
+        </div>
+      </div>
+    </SfSection>
+    <SfBanner
+      title="Download our application to your mobile"
+      subtitle="Fashion to Take Away"
+      image="/assets/homepage/bannerD.png"
+      class="banner-application sf-banner--left sf-banner--center desktop-only"
+    >
+      <template #title>
+        <h1 class="banner-application__title">
+          Download our application to your&nbsp;mobile
+        </h1>
+      </template>
+      <template #call-to-action>
+        <div>
+          <img
+            class="banner-application__download"
+            src="/assets/homepage/google.png"
+            alt=""
+          >
+          <img
+            class="banner-application__download"
+            src="/assets/homepage/apple.png"
+            alt=""
+          >
+        </div>
+      </template>
+    </SfBanner>
+  </div>
+</template>
+<script>
+import {
+  SfHero,
+  SfCallToAction,
+  SfSection,
+  SfCarousel,
+  SfProductCard,
+  SfImage,
+  SfBannerGrid
+} from '@storefrontui/vue';
+
+import { SfBanner } from 'src/themes/capybara/components/ui'
+import { isServer } from '@vue-storefront/core/helpers'
+import heroData from 'src/themes/capybara/assets/homepage/hero.json'
+
+export default {
+  name: 'HomePage',
+  data () {
+    return {
+      heroes: heroData
+    };
+  },
+  computed: {
+    newProducts () {
+      return this.$store.getters['homepage/getNewProducts']
+    }
+  },
+  async asyncData ({ store, route }) {
+    await store.dispatch('homepage/fetchNewProducts')
+  },
+  beforeRouteEnter (to, from, next) {
+    if (!isServer && !from.name) { // Loading products to cache on SSR render
+      next(vm => {
+        vm.$store.dispatch('homepage/fetchNewProducts')
+      })
+    } else {
+      next()
+    }
+  },
+  components: {
```

Comment:
we should have components above, rocommended way:

- name
- assets (mixins, components, directives, filters)
- props
- data
- computed
- methods
- lifecycle hooks
- render (if used)

source: [The Pope](https://github.com/vuejs/eslint-plugin-vue/issues/13#issuecomment-306128173)